app/controllers/api/v0/tags_controller.rb

```diff
@@ -3,19 +3,17 @@ module V0
     class TagsController < ApiController
       before_action :set_cache_control_headers, only: %i[index]
 
-      ATTRIBUTES_FOR_SERIALIZATION = %i[id name bg_color_hex text_color_hex short_summary badge_id].freeze
+      ATTRIBUTES_FOR_SERIALIZATION = %i[id name bg_color_hex text_color_hex].freeze
```

Comment:
This change makes it so that anyone consuming this API endpoint no longer has access to these attributes. Is this an intentional change?

Reply:
yep -- intentional here too 😄